### PR TITLE
Import documents

### PR DESCRIPTION
**Pull request by gabrielhase**
The README.md contains all the documenation.

### CODE REVIEW

File: lib/livingdocs_connector.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const _ = require('lodash')
+const axios = require('axios')
+const {mapMetadata} = require('../mappers/image_mapper')
+const documentMapper = require('../mappers/document_mapper')
+const mapToAbstractLivingdoc = require('../mappers/map_dpa_article_to_dpa_design')
+
+module.exports = {
```
Comment (meinradandermatt) [inline lib/livingdocs_connector.js:7]: It would be nice to have that in the sdk.

File: mappers/map_dpa_article_to_dpa_design.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      identifier: 'image',
+      id: `doc-${nanoid()}`,
+      content: {
+        image: image.image,
+        caption: image.caption,
+        source: image.source
+      }
+    })
+  }
+  const $ = cheerio.load(dpaArticle.article_html)
```
Comment (meinradandermatt) [inline mappers/map_dpa_article_to_dpa_design.js:42]: this is really difficult to read. Would be nice to have functions for each case.

File: lib/livingdocs_connector.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+const _ = require('lodash')
+const axios = require('axios')
+const {mapMetadata} = require('../mappers/image_mapper')
```
Comment (meinradandermatt) [inline lib/livingdocs_connector.js:3]: I would separate the Livingdocs API and the creation of the document

File: lib/transmogrifier.js [hunk 4]
```diff
@@ -0,0 +1,346 @@
+const {getComponentName} = require('./utils')
```
Comment (meinradandermatt) [inline lib/transmogrifier.js:1]: Would be nice to have the transmofrifier in separate library.

File: lib/livingdocs_connector.js [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    return {state: res.data.state, images: res.data.images}
+  },
+  importImages: async function (images) {
+    const config = {
+      headers: {Authorization: `Bearer ${process.env.LI_TOKEN}`}
+    }
+    const data = {
+      systemName: 'dpa-import-example',
+      // TODO for deployed versions change the host part
+      webhook: 'http://localhost:3000/dev/images-imported',
```
Comment (meinradandermatt) [inline lib/livingdocs_connector.js:55]: you could use env variables for the host part

File: lib/s3_helpers.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+module.exports = {
+  getArticlesFromBucket: async function (records) {
+    const getters = _.map(records, (record) => {
+      const bucket = record.s3.bucket.name
+      const key = record.s3.object.key
+
+      async function getArticle () {
+        const S3 = new AWS.S3({
+          s3ForcePathStyle: true,
+          endpoint: new AWS.Endpoint('http://localhost:4569')
```
Comment (meinradandermatt) [inline lib/s3_helpers.js:13]: Same here, would be nice to have this as config or env variable

**Review by meinradandermatt — COMMENTED** [review]
I have some inputs feel free to use it or merge without it ;-)

File: lib/livingdocs_connector.js (hunk 1 above)
Comment (gabrielhase) [reply lib/livingdocs_connector.js:7]: Currently, the SDK does not support POST request. I'd need some input from core devs first to change this.

File: lib/livingdocs_connector.js (hunk 3 above)
Comment (gabrielhase) [reply lib/livingdocs_connector.js:3]: Once, we have the import in the SDK, the API part will be in the SDK.

**Pull request merged**